build.sbt

```diff
@@ -39,7 +39,6 @@ lazy val root = Project("geotrellis", file("."))
   .settings(unidocProjectFilter in (ScalaUnidoc, unidoc) := inAnyProject)
 
 lazy val mdoc = project
```

Comment:
Could you skip it rom publish? `publish / skip := true` 
Should not mdoc be added into the [root aggregate project](https://github.com/locationtech/geotrellis/blob/57e366756aec5dc85b01e1ed9bc07173056bc899/build.sbt#L8)?

Reply:
Good call, done.